modules/auxiliary/client/telegram/send_message.rb

```diff
@@ -0,0 +1,55 @@
+# Content: Metasploit Module to send telegram message
+# Author: Ege Balcı| @egeblc | https://pentest.blog
+# Date: 07/2020
+
+require 'uri'
+
+class MetasploitModule < Msf::Auxiliary
+    def initialize
+        super(
+        'Name' => 'Telegram Message Client',
+        'Description' => 'This Module will send a telegram message to given chat ID with given bot token',
+        'Author' => 'Ege Balcı <egebalci@pm.me>',
```

Comment:
The `@` sign here needs to be replaced with `[at]` so that our backend parsers can correctly parse your email address. Also if you want to add your Twitter handle, you can do so by placing it as a comment on this line.

Reply:
Fixed with latest commits.